v2/controllers/egress_controller_test.go

```diff
@@ -469,4 +468,46 @@ var _ = Describe("Egress reconciler", func() {
 		Expect(saNS).To(HaveKey("default"))
 		Expect(saNS).To(HaveKey("egtest"))
 	})
+
+	It("should take over annotations when updated", func() {
```

Comment:
`It should take over annotations when updated` is not clear to me. I think this test checks if egress-controller doesn't remove the Spec.Template.Annotations of Deployment that might have been updated by clients like kubectl when it reconciles Deployment. Could you rephrase it?

Reply:
How about `It shouldn't remove annotations when reconciling deployment`?

Reply:
It would be better.